integrations/jenkins/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,43 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: job
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: .fullName | gsub(" "; "") | gsub("/"; "-") | ascii_downcase
+          title: .fullName
+          blueprint: '"jenkinsJob"'
+          properties:
+            jobName: .name
+            url: .url
+            jobStatus: '{"notbuilt": "created", "blue": "passing", "red": "failing"}[.color // "updated"]'
+            timestamp: .time
+            parentJob: .__parentJob
+  - kind: build
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: |
+            def parseURL:
+                  capture("^((?<scheme>[^:/?#]+):)?(//(?<authority>(?<domain>[^/?#:]*)(:(?<port>[0-9]*))?))?((?<path>[^?#]*))?(\\?(?<query>([^#]*)))?(#(?<fragment>(.*)))?");
+            
+            .url | parseURL | .path | gsub("%20"; "")  | gsub("/job/"; "/") | ascii_downcase | gsub("/"; "-") | gsub("^[/\\-]+|[/\\-]+$"; "")
```

Comment:
This just got the mapping way way way more complicated
if you can do something with jq dont force it please
clients need to understand it

Reply:
I understand you on this. Please check out this slack [thread](https://getport.slack.com/archives/D0696BJPNK1/p1703595840746549?thread_ts=1703164476.063059&cid=D0696BJPNK1) where I had raised my concern over this issue. It seemed like we had agreed on using a custom field of `__jobName` but the last [feedback](https://getport.slack.com/archives/D0696BJPNK1/p1703164476063059) went in a different direction of going back to jq.